docs/proposals/20250918-introduce_inferenceset_autoscaling.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Introduce a new InferenceSet CRD and Controller for scaling inference workloads automatically
+authors:
+  - "@andyzhangx"
+reviewers:
+  - "@Fei-Guo"
+  - "@rambohe-ch"
+  - "@zhuangqh"
```

Comment:
It would be nice to update this list to give credit to all the reviewers. please add @chewong here.